spec/unit/cli_spec.rb

```diff
@@ -660,6 +660,23 @@
   end
 
   describe "#error_handling" do
```

Comment:
Not related to your changes but seems this should actually be ".handle_error" ... maybe we can add it in an additional commit since it's such a small change